numbat-cli/src/main.rs

```diff
@@ -370,6 +370,15 @@ impl Cli {
                                 println!();
                             }
                             _ => {
+                                if let Some(keyword) = line.strip_prefix("help") {
```

Comment:
I was thinking about using `help <identifier>` again, and wondered if we should maybe introduce a new `info <identifier>` instead? I think it would reflect a bit better what the command does. And it would keep `help <topic>` free if we ever want to enhance the in-program help text for things like `help commands`, `help syntax` or similar.

What do you think?

Reply:
Sure, I like that suggestion.  I've just pushed a commit that renames the command to `info`